Iryna_Ramanets/Module_1_Java/Java_Fundamentals/HW2/src/com/epam/cdp/chef/interfaces/ICalculateCalories.java

```diff
@@ -0,0 +1,10 @@
+package com.epam.cdp.chef.interfaces;
+
+public interface ICalculateCalories {
```

Comment:
it's not really applicable name for interface. something like ICaloriesCalculatable should be better